esmvaltool/config-reformat-file.yml

```diff
@@ -10,6 +10,6 @@ input_dir: ./TEST_INPUT
 # Destination directory
 output_dir: ./TEST_OUTPUT
 # Dataset
-datasets: ['ESRL']
+datasets: ['DUMMY']
 # Variables
 variables: ['pr']
```

Comment:
There no need to specify the variable here, this is done within each script.

Reply:
ok cool, will remove

Reply:
actually var could be useful since we don't want to repeat the reformatting each time we add a new variable file?

Reply:
Yes, but then we will end up defining a configuration file for each dataset, or the user would need to edit the configuration file each time a reformat script is run. This also mean he would have to dig through the dataset and check which variable are needed.

If you set the variable (or variable list) directly in the reformat script the user does not need to care about anything (which is what he usually likes). You have to see the reformat scripts as black boxes: I download the data, I put them somewhere, I defined this somewhere as RAWOBS inputdir, I run the script. Done.